tutorials/time-series-modeling/Lab_05.md

```diff
@@ -0,0 +1,392 @@
+---
+title: Time Series Modeling
+description: This lab will go over the fundamentals of Time Series Modeling, including decomposition, autocorrelation and modeling historical changes.
+author: ghidora77, benamie
```

Comment:
Our publishing system can only handle one author, please remove the second and comma.

Reply:
Removed secondary author